### ISSUE TITLE
"sanitizing HTML stripped some content." since angular2 rc2

### ISSUE THREAD

**Issue by delike** [mattlewis92/angular2-bootstrap-confirm#14]
"WARNING: sanitizing HTML stripped some content."

This warning i got 5 times when i click on the Button to show the popup and the popup isn't showing :(

Testet in Safari and Chrome on OS X with angular2 rc2 and rc3.

At angular 2 rc1 it worked perfectly.

**Comment by mattlewis92** [comment]
That'll be because rc2+ added html escaping (similar to ng1's ng-sanitize module), I'm guessing you've got some unsafe html in the values you're passing to this directive. Please can you share some code that reproduces your problem? 😄

**Comment by delike** [comment]
Hi, thanks for your quick reply :)

i have some special character in the message attribute. no, without the special character there are no errors but also no confirm-dialog :(

here is my code:

`<div class="btn-group">
                                        <button
                                                class="btn btn-danger"
                                                mwl-confirm
                                                [title]="'Danger'"
                                                [message]="'Sure?'"
                                                placement="top"
                                                [confirmText]="'Yes'"
                                                [cancelText]="'No'"
                                                (confirm)="deleteMessage(message)"
                                                [(isOpen)]="isOpen">Del
                                        </button>
                                        <button class="btn btn-success" (click)="updateMessage(message)">Speichern</button>
                                    </div>`

**Comment by mattlewis92** [comment]
hmm there's nothing there that would cause it to break, what does your component annotation look like?

Are you specifying the providers and directives on it?
```
providers: [
    ConfirmOptions,
    {provide: Position, useClass: PositionService}
  ],
  directives: [
    Confirm
  ]
```

**Comment by delike** [comment]
yes
`
directives: [ROUTER_DIRECTIVES, MODAL_DIRECTVES, SELECT_DIRECTIVES, BUTTON_DIRECTIVES, Accordion, AccordionTab, Header, TextareaAutosizeDirective, Confirm],
    providers: [ConfirmOptions,
        provide(Position, { // this is required so you can use the bundled position service rather than rely on the `ng2-bootstrap` module
            useClass: PositionService
        })],
`

**Comment by delike** [comment]
yes i have "version": "0.5.0"

**Comment by mattlewis92** [comment]
I'm out of ideas, can you give me a full set of code that reproduces your issue? Maybe fork this repo and try and reproduce it on [the demo](https://github.com/mattlewis92/angular2-bootstrap-confirm/blob/master/demo/demo.ts). You can find dev setup instructions [here](https://github.com/mattlewis92/angular2-bootstrap-confirm#development)

**Comment by mattlewis92** [comment]
I gave this some more thought today and realised the issue is with `[(isOpen)]="isOpen"`, if you remove that from your code it should fix the problem. I will push a patch release as soon as I have some free time over the next few days 😄

**Comment by mattlewis92** [comment]
This should be fixed in 0.5.2 now, thanks for reporting!

**Comment by delike** [comment]
thanks! now its working 👍
